### PR TITLE
KEYCLOAK-14317 [sso73-dev] Stop using service ca from service account token

### PR DESCRIPTION
**Pull request by marun**
This change is required by [Bugzilla 1813894](https://bugzilla.redhat.com/show_bug.cgi?id=1813894). Inclusion of the service ca in token configmaps is discontinued in 4.5.

### CODE REVIEW

File: templates/sso73-x509-https.json [hunk 1]
```diff
@@ -369,6 +388,12 @@
                                 "secret": {
                                     "secretName": "sso-x509-jgroups-secret"
                                 }
+                            },
+                            {
+                                "name": "service-ca",
+                                "configMap": {
+                                    "configMapName": "${APPLICATION_NAME}-service-ca"
```
Comment (iankko) [inline templates/sso73-x509-https.json]: Replace `"configMapName"` with `"name"` like in the [RH-SSO 7.4 equivalent](https://github.com/jboss-container-images/redhat-sso-7-openshift-image/pull/145/files#r429224441) of the PR.

Current version returns
```
    error: DeploymentConfig.apps.openshift.io "sso" is invalid: [spec.template.spec.volumes[2].configMap.name: Required value, spec.template.spec.containers[0].volumeMounts[2].name: Not found: "service-ca"]
--> Failed
```
error. After the change the app is deployed from template successfully

File: templates/sso73-x509-mysql-persistent.json [hunk 2]
```diff
@@ -492,6 +511,12 @@
                                 "secret": {
                                     "secretName": "sso-x509-jgroups-secret"
                                 }
+                            },
+                            {
+                                "name": "service-ca",
+                                "configMap": {
+                                    "configMapName": "${APPLICATION_NAME}-service-ca"
```
Comment (iankko) [inline templates/sso73-x509-mysql-persistent.json]: Ditto here (replace with `"name"` field)

File: templates/sso73-x509-postgresql-persistent.json [hunk 3]
```diff
@@ -474,6 +493,12 @@
                                 "secret": {
                                     "secretName": "sso-x509-jgroups-secret"
                                 }
+                            },
+                            {
+                                "name": "service-ca",
+                                "configMap": {
+                                    "configMapName": "${APPLICATION_NAME}-service-ca"
```
Comment (iankko) [inline templates/sso73-x509-postgresql-persistent.json]: And finally here yet

**Review by iankko — CHANGES_REQUESTED** [review]
Replace all instances of `"configMapName"` field with `"name"` field. Once done, the template works OK / deploys the RH-SSO 7.3 pod correctly (verified by testing)

File: templates/sso73-x509-https.json (hunk 1 above)
Comment (marun) [reply templates/sso73-x509-https.json]: Done

File: templates/sso73-x509-mysql-persistent.json (hunk 2 above)
Comment (marun) [reply templates/sso73-x509-mysql-persistent.json]: Done

File: templates/sso73-x509-postgresql-persistent.json (hunk 3 above)
Comment (marun) [reply templates/sso73-x509-postgresql-persistent.json]: Done

**Comment by marun** [comment]
@iankko Updated, PTAL

**Review by mhajas — APPROVED** [review]
LGTM

**Comment by marun** [comment]
@hmlnarik @mhajas Any blocker to this being merged? As per @iankko's [comment]( https://github.com/jboss-container-images/redhat-sso-7-openshift-image/pull/146#pullrequestreview-416903264) the change has been tested.

**Pull request merged**